files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -124,15 +125,35 @@ def update_filter_metadata(metadata_to_update):
                 mds_res = mds_res[cedar_record_id]
                 mds_cedar_register_data_body = {}
                 mds_discovery_data_body = {}
+                mds_clinical_trials = {}
                 if mds_res["_guid_type"] == "discovery_metadata":
                     print("Metadata is already registered. Updating MDS record")
                 elif mds_res["_guid_type"] == "unregistered_discovery_metadata":
                     print("Metadata is has not been registered. Registering it in MDS record")
-                    continue
 
-                pydash.merge(mds_discovery_data_body, mds_res["gen3_discovery"], cedar_record)
+                if "clinicaltrials_gov" in cedar_record:
+                    mds_clinical_trials = cedar_record["clinicaltrials_gov"]
+                    del cedar_record["clinicaltrials_gov"]
+
+                pydash.merge(mds_res["gen3_discovery"]["study_metadata"], mds_res["gen3_discovery"]["study_metadata"], cedar_record)
```

Comment:
does this work? I think there is a slight difference in format between `mds_res["gen3_discovery"]["study_metadata"]` and `cedar_record`

For example: for appl_id, the field name in `mds_res["gen3_discovery"]["study_metadata"]` is `metadata_location.nih_application_id`, but in `cedar_record` it is `Metadata Location.nih_application_id` 🤔 

Metadata Location.nih_application_id